examples/flutter_view/android/app/src/main/java/com/example/flutter/MainActivity.java

```diff
@@ -0,0 +1,80 @@
+package com.example.flutter;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.TextView;
+import io.flutter.view.FlutterMain;
+import io.flutter.view.FlutterView;
+
+public class MainActivity extends AppCompatActivity {
+    private FlutterView flutterView;
+    private int counter;
+    private static final String CHANNEL = "increment";
+    private static final String EMPTY_MESSAGE = "";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        FlutterMain.ensureInitializationComplete(getApplicationContext(), null);
```

Comment:
The `null` argument implies that information about profiling mode etc is not transferred to the Dart VM.